crates/ruff_linter/src/rules/flake8_type_checking/rules/typing_only_runtime_import.rs

```diff
@@ -490,7 +506,12 @@ fn is_exempt(name: &str, exempt_modules: &[&str]) -> bool {
 }
 
 /// Generate a [`Fix`] to remove typing-only imports from a runtime context.
-fn fix_imports(checker: &Checker, node_id: NodeId, imports: &[ImportBinding]) -> Result<Fix> {
+fn fix_imports(
+    checker: &Checker,
+    node_id: NodeId,
+    imports: &[ImportBinding],
+    add_future_import: bool,
```

Comment:
Nit: I'm somewhat inclined to store on the binding whether it needs a from future import, given that we iterate all bindings anyway to find `at`